spec/factories/item.rb

```diff
@@ -0,0 +1,89 @@
+# frozen_string_literal: true
+
+# frozen_string_literal => true
+
```

Comment:
```suggestion
# frozen_string_literal: true
```

Reply:
oops :)

Reply:
why doesn't it letting me to commit the suggestion 🤔

Reply:
🤷‍♂ 
maybe because I already approved?

did like the new multi-line suggestion though 💯

Reply:
ahhh yeah I guess so, I will commit it with the other changes 👍

Reply:
oddly my other suggestion seems commit-able 🤷‍♂

Reply:
yes 🤷‍♀  weird

Reply:
ok committed the change